### PR TITLE
Start using Phenix

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer:

/cc @bquorning @grosser 

### Steps to merge
* [ ] :+1: from the team

### Risks
* None

### CODE REVIEW

File: gemfiles/rails4.0.gemfile.lock [hunk 1]
```diff
@@ -1,59 +0,0 @@
-PATH
```
Comment (bquorning) [inline gemfiles/rails4.0.gemfile.lock]: This removal warrants at least a separate commit.

**Comment by pschambacher** [comment]
@bquorning :haircut:

File: test/global_uid_test.rb [hunk 2]
```diff
@@ -1,92 +1,18 @@
...
-  def self.up
-    create_table :without_global_uids, :use_global_uid => false do |t|
-      t.string  :description
-    end
-  end
-
-  def self.down
-    drop_table :without_global_uids, :use_global_uid => false
+describe GlobalUid do
+  before do
```
Comment (grosser) [inline test/global_uid_test.rb]: this used to be a 1-time setup and now it's in every before ?

File: global_uid.gemspec [hunk 3]
```diff
@@ -19,6 +19,7 @@ Gem::Specification.new 'global_uid', '3.3.2' do |s|
   s.add_development_dependency('mocha')
   s.add_development_dependency('bump')
   s.add_development_dependency('wwtd', '>= 0.5.3')
+  s.add_development_dependency('phenix')
 
-  s.files = `git ls-files lib`.split("\n")
+  s.files = Dir.glob('lib/**/*')
```
Comment (grosser) [inline global_uid.gemspec]: - `ls-files` prevents shipping temp files and other junk
 - shipping the Readme might be nice too

File: test/global_uid_test.rb (hunk 2 above)
Comment (grosser) [inline test/global_uid_test.rb]: never mind ... misread that, was always on each test case

File: test/test_helper.rb [hunk 4]
```diff
@@ -14,8 +15,12 @@
...
 GlobalUid::Base.extend(GlobalUid::ServerVariables)
 
-yaml = YAML.load(IO.read(File.dirname(__FILE__) + "/config/database.yml"))
-ActiveRecord::Base.configurations = yaml
+Phenix.configure do |config|
+  config.database_config_path = File.join(File.dirname(__FILE__), "config/database.yml")
+end
+
+Phenix.rise!(with_schema: false)
 ActiveRecord::Base.establish_connection(:test)
```
Comment (grosser) [inline test/test_helper.rb]: should connecting be handled inside of phenix ? / can we avoid the duplicate .rise! ?

File: test/test_helper.rb [hunk 5]
```diff
@@ -14,8 +15,12 @@
 }
 GlobalUid::Base.extend(GlobalUid::ServerVariables)
 
-yaml = YAML.load(IO.read(File.dirname(__FILE__) + "/config/database.yml"))
-ActiveRecord::Base.configurations = yaml
+Phenix.configure do |config|
+  config.database_config_path = File.join(File.dirname(__FILE__), "config/database.yml")
```
Comment (grosser) [inline test/test_helper.rb]: could configure with `schema_path nil` and then schema will not be loaded ?

**Comment by grosser** [comment]
:+1:

**Pull request merged**